recipe/meta.yaml

```
@@ -20,20 +20,21 @@ build:
     # SO name changes at minor rev bumps.
     - {{ pin_subpackage('sleef', max_pin='x') }}
   skip: true  # [win and vc<14]
-  ignore_run_exports_from:
-    - openssl
 
 requirements:
   build:
     - {{ compiler('c') }}
     - cmake
     - ninja        # [unix]
```

```suggestion
    - ninja-base        # [unix]
```
How about `ninja-base` instead of `ninja`?